### ISSUE TITLE
Add example image to README

### ISSUE THREAD

**Issue by cbenhagen** [nank1ro/flutter-shadcn-ui#76]
The README would benefit from an example how beautiful an app using `Flutter ShadcnUI` could look like. 

<img width="1370" alt="Screenshot 2024-06-16 at 08 57 37" src="https://github.com/nank1ro/flutter-shadcn-ui/assets/814785/a09894cb-fdde-4cdf-8a9b-5f703370d0bf">
<img width="1306" alt="Screenshot 2024-06-16 at 08 58 15" src="https://github.com/nank1ro/flutter-shadcn-ui/assets/814785/16b1ab23-e6d6-4366-b107-43b9ffa11b60">

**Comment by nank1ro** [comment]
Yes I know, but not all the components are ready. So the pages you've shown is not guaranteed to be rendered in the same way.
Don't know If I need to wait for the v1 with all the components implemented and trying to implement these pages

**Comment by cbenhagen** [comment]
I think the best approach would be to extend the example so it not only has pages for all the components but also some example apps like in the screenshot which use the components available today. That way we could provide a flutter web version for people to play with as well as take screenshots from there for the README which only show what is actually implemented.

**Comment by nextonon** [comment]
Ciao 
Very lacking examples, came to Flutter from Vue and that to understand need an example of at least a simple screen with a couple of widgets and components
